### ISSUE TITLE
Question about model assessment

### ISSUE THREAD

**Issue by XinyuSun** [FingerRec/BE#2]
In section 4.5 **How does Background Erasing Work**, the author first trained an I3D model with *static* video generated from HMDB51 and draws classification top-1 acc of each class. But in our experiments, we found the validation acc of each class is similar due to the small amount of validation set.

Thus, we doubt whether the classification top-1 acc results in fig.4 are reported from the train set or validation set? They are evenly distributed on the horizontal axis. It is appreciated if more information about the experiment in section 4.5 is provided.

- Here is our experiment result, where the average top-1 acc is `15.82`.
```bash
class          val acc  train acc
0	===>	0.00,	97.14
47	===>	0.00,	93.55
43	===>	0.00,	100.00
38	===>	0.00,	91.89
37	===>	0.00,	76.47
36	===>	0.00,	92.50
27	===>	0.00,	69.70
20	===>	0.00,	94.12
19	===>	0.00,	97.06
14	===>	0.00,	81.08
13	===>	0.00,	96.88
12	===>	0.00,	65.71
50	===>	0.00,	84.21
1	===>	0.00,	83.87
4	===>	0.00,	94.59
11	===>	0.00,	84.38
31	===>	6.67,	87.80
33	===>	6.67,	91.67
39	===>	6.67,	85.71
24	===>	6.67,	94.74
5	===>	6.67,	93.33
21	===>	6.67,	93.33
8	===>	6.67,	85.37
41	===>	6.67,	96.15
17	===>	6.67,	78.38
16	===>	6.67,	83.33
48	===>	6.67,	84.09
7	===>	6.67,	94.29
45	===>	6.67,	90.00
42	===>	13.33,	81.58
26	===>	13.33,	100.00
28	===>	13.33,	91.18
49	===>	13.33,	88.89
46	===>	13.33,	81.82
40	===>	13.33,	85.71
32	===>	20.00,	83.33
6	===>	20.00,	93.94
34	===>	20.00,	94.74
23	===>	20.00,	90.62
22	===>	20.00,	80.00
29	===>	20.00,	100.00
25	===>	26.67,	94.87
9	===>	40.00,	93.94
3	===>	40.00,	93.55
10	===>	40.00,	85.71
2	===>	46.67,	93.55
44	===>	46.67,	83.78
35	===>	53.33,	94.59
18	===>	53.33,	93.94
15	===>	53.33,	100.00
30	===>	66.67,	93.55
```

**Comment by FingerRec** [comment]
Hi xingyu:

Thanks for your interest in our work.  Some points need to be noticed:
(1). Use 3 splits and get average result.
(2). Train 3D CNN with 1 static video in normal CE loss, the train acc shouldn't be close to 100 with static video.

Yours,
Awiny

**Comment by XinyuSun** [comment]
Hi Jinpeng, thanks for replying.

There are some questions about the detail:
- How many epochs have you trained on each split of HMDB51?
- The average validation accuracy.

**Comment by FingerRec** [comment]
Hi Xinyu,

We trained the static video classification about 100 epochs.
Sorry for late reply.  The avg top-1 acc on HMDB51 should be 29.7\% +- 1\%.
As we find random initialization affects the prediction result, compute the evaluation multiple times and average result.

Yours,
Awiny
